utils/clubs.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const request = require('request');
+
+const mongoose = require('mongoose');
+var db = mongoose.connect(process.env.MONGODB_URI, {useNewUrlParser: true, useUnifiedTopology: true});
+
+const Club = require('../model/club');
+const Member = require('../model/member');
+const ftw = require('./ftw');
+
+function callName (senderId, callback) {
+    request({
+        url: "https://graph.facebook.com/v4.0/" + senderId,
+        qs: {
+            access_token: process.env.PAGE_ACCESS_TOKEN,
+            fields: "name"
+        },
+        method: "GET"
+    }, function (err, response, body) {
+        if (err)
+            console.log("Error getting user's name: " +  err);
+        else {
+            var bodyObj = JSON.parse(body);
+            var name = bodyObj.name;
+            callback(senderId, name);
+        }
+    });
+}
+
+// only person who can modify club metadata is club owner e.g. owner's sender id
```

Comment:
Use page's id instead.

(fyi page id can be found in the `body` object [here](https://github.com/wilsonh1/PalindromeDuck2/blob/master/index.js#L17))